test/e2e/specs/demo-content-with-keyboard.test.js

```diff
@@ -0,0 +1,345 @@
+/**
+ * Node dependencies
+ */
+import path from 'path';
+import fs from 'fs';
+import os from 'os';
+import uuid from 'uuid/v4';
+
+/**
+ * Internal dependencies
+ */
+import {
+	newPost,
+	pressWithModifier,
+	META_KEY,
+	pressTimes,
+	//getEditedPostContent,
+} from '../support/utils';
+
+async function uploadImageInTheMediaLibrary( assetFileName ) {
+	await page.waitForSelector( '.media-modal input[type=file]' );
+	const inputElement = await page.$( '.media-modal input[type=file]' );
+	const testImagePath = path.join( __dirname, '..', 'assets', assetFileName );
+	const filename = uuid();
+	const tmpFileName = path.join( os.tmpdir(), filename + '.png' );
+	fs.copyFileSync( testImagePath, tmpFileName );
+	await inputElement.uploadFile( tmpFileName );
+	await page.waitForSelector( `.media-modal li[aria-label="${ filename }"]` );
+}
+
+async function navigateToNextRegion() {
+	await page.keyboard.down( 'Alt' );
+	await page.keyboard.down( 'Shift' );
+	await page.keyboard.type( 'n' );
+	await page.keyboard.up( 'Shift' );
+	await page.keyboard.up( 'Alt' );
+}
+
+async function createBlockAfter() {
+	await page.keyboard.down( META_KEY );
+	await page.keyboard.down( 'Alt' );
+	await page.keyboard.type( 'y' );
+	await page.keyboard.up( 'Alt' );
+	await page.keyboard.up( META_KEY );
+}
+
+jest.setTimeout( 1000000 );
+
+describe( 'Demo content post', () => {
+	beforeAll( async () => {
+		await newPost();
+	} );
+
+	it( 'should be created with keyboard', async () => {
+		// Title of the page
+		await page.keyboard.type( 'Welcome to the Gutenberg Editor' );
+
+		// Creating the cover image
+		await page.keyboard.press( 'Enter' );
+		await page.keyboard.type( '/cover' );
+		await page.keyboard.press( 'Enter' );
+
+		// Apply center alignment (wide alignment is not available in the default theme)
+		await pressWithModifier( 'Alt', 'F10' );
+		await pressTimes( 'Tab', 2 );
+		await page.keyboard.press( 'Enter' );
+
+		// Upload the image
+		await pressTimes( 'Tab', 4 ); // We should have a way to focus the block content
+		await page.keyboard.press( 'Enter' );
+		await uploadImageInTheMediaLibrary( 'cover-1.jpg' );
+		await page.click( '.media-modal button.media-button-select' ); // validating the media gallery is not easy with keyboard
```

Comment:
Not easy or impossible?

Reply:
Possible but you need to count "tabs" which is not great and break easily. There's probably a way to do it which is "tab + test if we reached the correct button"

Reply:
Inserting media was a bit painful in my manual testing, I had to go through all the fields before I could insert.

Reply:
Would:

```suggestion
		await tabUntilElementIsActive( '.media-modal button.media-button-select' );
```

work here? If so it'd be great, but if not: okay. 🤷‍♂️ A follow-up issue to track it would be good though.

Reply:
It would work yet but I consider `tabUntilElementIsActive` not an ideal solution. In general we should have some direct ways to access these buttons (like aria-regions in the medial library for instance).

But I'll update as it's slightly better.